graphx/src/main/scala/org/apache/spark/graphx/VertexRDD.scala

```diff
@@ -128,7 +128,7 @@ abstract class VertexRDD[VD](
    *
    * @param other the other RDD[(VertexId, VD)] with which to diff against.
    */
-  def diff(other: RDD[(VertexId, VD)]): VertexRDD[VD]
+  def diff(other: RDD[(VertexId, VD)]): VertexRDD[VD] = ???
```

Comment:
I see. In that case the exclusion is the right thing, then. I'll update the code.